.github/workflows/test.yml

```diff
@@ -0,0 +1,20 @@
+name: Parser compilation and tests
+
+on: [push, pull_request]
+
+jobs:
+  check_compilation_with_cl:
+    name: Parser compilation using MSVC cl.exe
+    runs-on: 'windows-latest'
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ilammy/msvc-dev-cmd@v1.5.0
+
+      - name: Install and prepare Neovim
+        run: |
+          bash ./scripts/ci-install-windows-latest.sh
+          $env:Path += "C:\\tools\\neovim\\Neovim\\bin\\"
```

Comment:
maybe it requires a `refreshenv`?

Reply:
currently went with complete path to `nvim.exe`, but seems there are some failures for some languages and also `gcc` appeared in the wild :)

Reply:
okay so issue was with those "special" ones with location specified in parser config